src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -150,8 +150,9 @@ typedef struct MPIDI_OFI_deferred_am_isend_req {
     MPI_Datatype datatype;
     MPIR_Request *sreq;
     bool need_packing;
-
     MPI_Aint data_sz;
+    int vni_src;
+    int vni_dst;
```

Comment:
Am I missing something or does this commit message have nothing to do with the change? I’m not sure what the change does…

Reply:
Yep, it is a left-over commit from rebasing. Now merged with another commit (the one to "enable per-vni active message")